Osteppy/src/server.js

```diff
@@ -35,182 +100,124 @@ app.use(morgan('dev'));
 
 // 3rd party middleware
 app.use(cors({
-	exposedHeaders: config.corsHeaders
+  exposedHeaders: config.corsHeaders,
 }));
 
 app.use(bodyParser.urlencoded({
-  extended: true
+  extended: true,
 }));
 
 /** Reached by Slack API */
 
 // Slash command for submitting EOD's
 app.post('/eod', (req, res) => {
-    const slack_request = req.body;
-
-	const slack_response = {
-		"response_type": "in_channel",
-		"text": `:checkered_flag: EOD was submitted by *${slack_request.user_name}*`,
-		"attachments": [
-			{
-				"text": `${slack_request.text}`
-			}
-		]
-    };
-
-	axios.post(slack_request.response_url, slack_response).then(() => {
-		let report_data = JSON.parse(fs.readFileSync(data_file, 'utf8'));
-		report_data[slack_request.user_name] = {
-			'time': new Date(),
-			'text': slack_request.text,
-			'channel': slack_request.channel_name 
-		};
-
-		fs.writeFileSync(data_file, JSON.stringify(report_data), 'utf8');
-
-    }).catch(error => {
-        console.log("error: " + error);
-	});
-    
-    // Remove RA's name from EOD reminder list
-    readRAs();
-    submitEOD(slack_request.user_name);
-    writeRAs();
-    printRAs();
-	res.status(200).send();
+  const slackRequest = req.body;
+
+  const slackResponse = {
+    response_type: 'in_channel',
+    text: `:checkered_flag: EOD was submitted by *${slackRequest.user_name}*`,
+    attachments: [
+      {
+        text: `${slackRequest.text}`,
+      },
+    ],
+  };
+
+  axios
+    .post(slackRequest.response_url, slackResponse)
+    .then(() => EOD.submit(slackRequest.user_name, {
+      time: new Date(),
+      text: slackRequest.text,
+      channel: slackRequest.channel_name,
+    })).catch((error) => {
+      console.log(`error: ${error}`);
+    });
+
+  res.status(200).send();
 });
 
 // Slash command for checking who have yet to submit EOD's
 app.post('/eod_left', (req, res) => {
```

Comment:
We could potentially group the requests e.g. `/eod`, `/eod/left`, `/eod/reminder/script_status`, `/eod/reminder/time` instead of  `/eod/eod_left`, `check_js_script` etc. (will need to update slack app configs then)